client.lua

```diff
@@ -39,11 +39,13 @@ RegisterNetEvent('QBCore:Client:OnPlayerUnload', function()
     TriggerServerEvent('qbx_npwd:server:UnloadPlayer')
 end)
 
--- Handles state when PlayerData is changed. We're just looking for inventory updates.
-RegisterNetEvent('QBCore:Player:SetPlayerData', function()
-    doPhoneCheck()
+AddEventHandler('ox_inventory:itemCount', function(itemName, totalCount)
+    if itemName == "phone" then
```

Comment:
Please invert this if statement and adjust so it checks the whole `PhoneList` instead of one item. Also an extra is to send the totalCount to `doPhoneCheck` so this doesn't need to be checked again